main/lsp/LSPIndexer.cc

```diff
@@ -68,53 +69,7 @@ void LSPIndexer::computeFileHashes(const vector<shared_ptr<core::File>> &files,
         return;
     }
 
-    Timer timeit(config->logger, "computeFileHashes");
```

Comment:
Note: Moved to pipeline so that it can be used by minimum realmain, which lacks LSP.

This is actually the logical home for this logic anyway!